hiddenlayer/services/ModelScanService.ts

```diff
@@ -7,23 +7,22 @@ import { NodeJsClient } from '@smithy/types';
 
 import { BlobServiceClient } from '@azure/storage-blob';
 
-import { SensorApi, ModelScanApi, Model, Configuration, ModelSupplyChainApi, ScanReportV3, ScanReportV3StatusEnum, Sarif210 } from "../../generated";
-import AdmZip from 'adm-zip';
+import { SensorApi, Configuration, ModelSupplyChainApi, ScanReportV3, ScanReportV3StatusEnum, Sarif210 } from "../../generated";
 import { sleep } from './utils';
 import { ModelService } from './ModelService';
 import "../extensions/ModelSupplyChainApiExtensions";
 
 export class ModelScanService {
     readonly sensorApi: SensorApi;
-    readonly modelScanApi : ModelScanApi;
+    //readonly modelScanApi : ModelScanApi;
     readonly modelSupplyChainApi: ModelSupplyChainApi;
     readonly modelService: ModelService;
     readonly isSaaS: boolean;
 
     constructor(isSaaS: boolean, config: Configuration) {
         this.isSaaS = isSaaS;
         this.sensorApi = new SensorApi(config);
-        this.modelScanApi = new ModelScanApi(config);
+        //this.modelScanApi = new ModelScanApi(config);
```

Comment:
If this isn't needed any more, let's just remove it.
```suggestion
        //this.modelScanApi = new ModelScanApi(config);
```